src/utils.ts

```diff
@@ -135,3 +137,33 @@ export const getActiveSectionsWithData = (
 
   return activeSectionsWithData;
 };
+
+export const handleSectionSearchTermHighlights = (
+  query: string,
+  sections: UserDefinedSection[]
+) => {
+  // Escape query from special characters
+  function escapeRegExp(string: string) {
+    return string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
+  }
+
+  const regex = new RegExp(`(${escapeRegExp(query)})`, 'gi');
+
+  sections.forEach((section: UserDefinedSection) => {
+    if (section?.displaySearchTermHighlights) {
+      const sectionName = section?.displayName || section?.identifier;
+
+      document
+        .querySelector(`.${sectionName?.split(' ')?.join('.')}`)
```

Comment:
I think generally, if possible we should try to avoid using querySelector. 

Do you think it would be possible to instead have this logic live in the SectionItem.tsx file, modify each item as they come?

That way we can work directly with an autocomplete value in React, not have to worry about sanitizing html.

Reply:
@stanlp1 Yes, that's what I did locally. I was waiting to confirm with the rest of the team, but seems like this is the right approach 👍